rpc/client.go

```diff
@@ -16,18 +26,34 @@ import (
 )
 
 type UploadReportOptions struct {
-	Report          *poplar.Report
-	ClientConn      *grpc.ClientConn
-	SerializeUpload bool
-	DryRun          bool
+	Report             *poplar.Report
+	ClientConn         *grpc.ClientConn
+	SerializeUpload    bool
+	AWSSecretKey       string
+	AWSAccessKey       string
+	ResultsHandlerHost string
+	ResultType         string
```

Comment:
I think we can just hardcode the `ResultType` as `cedar-report`. I don't see us using this function to ever send anything else.